vagrant/Makefile

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/make -f
+
+# mkfile_path is absolute path of this file
+# The intention is to be able to run this file from any location
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+# root_dir is a root directory of the project (github repo)
+root_dir := $(dir $(abspath $(mkfile_path)/..))
+helm_path = ${root_dir}deploy/helm/sumologic
+k8s_path = ${root_dir}vagrant/k8s
+dashboards_path = 
+
+# Collector configuration
+cluster = microk8s
+endpoint = http://receiver-mock.receiver-mock:3000/terraform/api/
+access_id = dummy
+access_key = dummy
+name = collection
+namespace = sumologic
+
+# Collector section
+## Collector upgrade
+upgrade:
+	${mkfile_path} \
+	  patch-context \
+	  remove-tmp \
+	  apply-namespace \
+	  apply-receiver-mock \
+	  create-secrets \
+	  grafana-dashboards \
+	  helm-dependencies \
+	  helm-upgrade \
+	  apply-service-monitors
```

Comment:
Since this is a `Makefile` we need tabs but why mix it with spaces here instead of just using 2 tabs?

Reply:
Indentation in the terminal looks better with two spaces instead of eight. I tried to omit tabs but it's not possible 😞 
@perk-sumo thoughts?

Reply:
I'd go with tabs to make it consistent across whole Makefile file.